blockchain/blockchain.go

```diff
@@ -891,6 +891,9 @@ func (bc *BlockChain) Stop() {
 		_ = triedb.TrieNodeCache().Close()
 	}
 
+	// reset hardForkBlockNumberConfig to nil
+	fork.ResetHardForkBlockNumberConfig()
```

Comment:
Do we need to clear it? `Stop()` closes resources used in `Blockchain` but doesn't clear configuration.

Reply:
I added this line because I thought `Stop` method is symmetric with `NewBlockchain`. In NewBlockchain hardForkBlockNumber is configured, so i reset it in Stop().. btw, this part is moved to new PR #1012.

Reply:
I'll reconsider it about this one more time, thanks!